### DISCUSSION TITLE
Can't install edge as package missing

### DISCUSSION THREAD

**Discussion by badjames** [microsoft/wslg#309] [Q&A]
According to the instructions at https://github.com/microsoft/wslg, I can install edge with:

sudo curl https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/microsoft-edge-dev_91.0.852.0-1_amd64.deb -o /tmp/edge.deb
sudo apt install /tmp/edge.deb -y

However, on the second command I get:

> sudo apt install /tmp/edge.deb -y
Reading package lists... Error!
E: Invalid archive signature
E: Internal error, could not locate member control.tar.{zstlz4gzxzbz2lzma}
E: Could not read meta data from /tmp/edge.deb
E: The package lists or status file could not be parsed or opened.

Apparently, the package no longer exists at:

https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/microsoft-edge-dev_91.0.852.0-1_amd64.deb

How do I now install Edge?

**Comment by onomatopellan** [comment]
Download with `wget` the version you want from [here.](https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/)
```
wget https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/microsoft-edge-dev_93.0.916.1-1_amd64.deb
sudo apt install ./microsoft-edge-dev_93.0.916.1-1_amd64.deb
```
